### PR TITLE
docs: Layout (QCol/QRow) docs

### PR DESCRIPTION
**Pull request by supra-modum**

### CODE REVIEW

File: docs/components/Layout.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+<iframe height="340" style="width: 100%;" scrolling="no" frameborder="no" src="/qui-max/Layout/Layout.html"></iframe>
+
+## Examples
+
+Default view code example below:
+
+```vue
+
+<div class="block">
+<q-row class="block__row">
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:17]: fixed

File: docs/components/Layout.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+```
+
+## QCol Props
+
+`QCol` uses these props
+
+```js
+const args = {
+    tag: 'div',
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:48]: fixed

File: docs/components/Layout.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+<iframe height="340" style="width: 100%;" scrolling="no" frameborder="no" src="/qui-max/Layout/Layout[qcol].html"></iframe>
+
+Below is an example of sample layout:
+
+```vue
+
+<q-row class="block__row">
+<q-col
+    class="block__col"
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:62]: fixed

File: docs/components/Layout.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+const args = {
+    tag: 'div',
+    cols: 4,
+    offset: 3
+};
+```
+
+<iframe height="340" style="width: 100%;" scrolling="no" frameborder="no" src="/qui-max/Layout/Layout[qcol].html"></iframe>
+
+Below is an example of sample layout:
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:56]: removed

File: docs/components/Layout.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    :cols="args.cols"
+    :offset="args.offset">
+  <div class="block__content">Lorem ipsum dolor</div>
+</q-col>
+</q-row>
+```
+
+### tag
+
+Custom element tag.
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:73]: fixed

File: docs/components/Layout.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+- Type: `String`
+- Default: `div`
+
+### cols
+
+Number of column the grid cols.
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:80]: fixed

File: docs/components/Layout.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+`auto || [1-12]`
+
+- Type: `String` or `Number`
+- Default: `auto`
+
+### offset
+
+Number of spacing on the left side of the grid.
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:89]: fixed

File: docs/components/Layout.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+<q-col class="block__col" cols="2">
+  <div class="block__content">2</div>
+</q-col>
+</q-row>
+```
+
+### tag
+
+Custom element tag.
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:125]: fixed

File: docs/components/Layout.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+### alignV
+
+Vertical alignment of flex layout: `['start', 'end', 'center', 'baseline', 'stretch']`
+
+- Type: `String`
+- Default: `null`
+
+### alignH
+
+Horizontal alignment of flex layout: `['start', 'end', 'center', 'between', 'around']`
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:139]: fixed

File: docs/components/Layout.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+
+### tag
+
+Custom element tag.
+
+- Type: `String`
+- Default: `div`
+
+### alignV
+
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:131]: fixed

File: docs/components/Layout.md [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+## QRow Props
+
+`QRow` uses these props
+
+
+```vue
+
+<q-row
+    class="block__row"
```
[the remark this answers is not in the document]
Comment (supra-modum) [reply docs/components/Layout.md:104]: fixed

**Review by Tim152 — APPROVED** [review]

**Pull request merged**